src/partials/header.html

```diff
@@ -0,0 +1,20 @@
+<div class="header-container">
+    <div class="header">
+        <nav class="header-nav">
+            <a href="#" class="header-nav__homePage-btn"><p>FILMOTEKA</p></a>
+                <svg class="header-nav__mobile-menu-icon" width="0" height="0" viewBox="0">
```

Comment:
W konsoli wywala błąd związany z tagiem svg: Unexpected end of attribute. Expected number, "0".

Atrybut viewBox określa, która część obrazu zostanie wyświetlona i w jakiej skali.
viewBox="x y width height"
Wartości x i y sterują przesunięciem od początku układu współrzędnych okna widoku i wyświetlają tylko część obrazu. Wartości width i height sterują skalowaniem.

Reply:
zmienione